src/servers/grpc_server.cc

```diff
@@ -249,14 +252,13 @@ GRPCServer::~GRPCServer()
 
 Status
 GRPCServer::Create(
-    InferenceServer* server, uint16_t port,
+    InferenceServer* server, int32_t port,
     std::unique_ptr<GRPCServer>* grpc_server)
 {
-  // DLIS-162 - provide global defaults and cli overridable options
   g_Resources = std::make_shared<AsyncResources>(
-      server,  // InferenceServer*,
-      1,       // infer threads
-      1        // mgmt threads
+      server, /* InferenceServer*, */
```

Comment:
Closer, don't comment the server argument, only constants. I'm surprised the auto-formatter didn't but these on one line.